style.css

```diff
@@ -76,14 +69,239 @@ h2 {
   color: #7f8cff;
 }
 
+/* Works section */
+.works-container,
+.multi-post-container {
+  display: grid;
+  grid-template-rows: auto;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.card-one {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+}
+
+.card-two {
+  margin-left: 25px;
+}
+
+.three-elements-row {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 75px;
+  padding: 20px 0;
+}
+
+.three-elements {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+}
+
+.tonic-title,
+.multi-post-title,
+.stories-title {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172d4d;
+}
+
+.canopy-title {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.back-end-title,
+.year-title {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.circle {
+  width: 8px;
+  height: 8px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.daily-selection-paragraph {
+  height: 80px;
+}
+
+.categories {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  gap: 8px;
+  margin: 10px 10px 0 6px;
+}
+
+.button1 {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 7px;
+  width: 126px;
+  height: 48px;
+  margin: 20px 0 80px 0;
+}
+
+.button2 {
+  font-family: "Poppins", arial, helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 7px;
+  width: 182px;
+  height: 48px;
+  margin: 20px 0 80px 0;
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. Please reduce your bottom margin from 80px to 20px and have a look at the below-attached image and table.

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-55-38](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/40948a08-411b-42b4-bd5f-7740b6992235)

</td>
<td>

![Screenshot from 2023-06-10 17-54-56](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/5d10129e-72ac-4801-b6e5-35a2da1878d2)

</td>
</tr>
</table>

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)